navigation/navigation-common/build.gradle

```diff
@@ -53,15 +53,14 @@ androidXMultiplatform {
     sourceSets {
         commonMain {
             dependencies {
-                api "org.jetbrains.compose.annotation-internal:annotation:1.7.1"
-                api "org.jetbrains.compose.collection-internal:collection:1.7.1"
-                api project(":core:core-bundle")
-                api project(":core:core-uri")
+                api project(":annotation:annotation")
+                api project(":collection:collection")
```

Comment:
Let's drop our stubs in navigation, we can use androidx.* directly here

Reply:
done